server/container_create.go

```diff
@@ -470,6 +470,7 @@ func (s *Server) CreateContainer(ctx context.Context, req *pb.CreateContainerReq
 		recordError(operation, err)
 	}()
 	logrus.Debugf("CreateContainerRequest %+v", req)
+	logrus.Debugf("Attempting to create container: %s", translateLabelsToDescription(req.GetConfig().GetLabels()))
```

Comment:
We need the traces to be always visible so use a log level that is visible by default. I think Infof should be visible without turning on debug.

Reply:
@mrunalp Infof still needs --log-level info when calling crio. Now, it's easier to see the traces if they're info and differentiated from all of the debugging info, so I can leave it. but it won't be default.